### PR TITLE
PSMDB-1559 Release notes 7.0.15-9

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   docs/release_notes/7.0.15-9.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release_notes/7.0.15-9.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Percona Server for MongoDB 7.0.15-9 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB Community Edition.
+
+Percona Server for MongoDB 7.0.15-9 includes the improvements and bug fixes of [MongoDB 7.0.15 Community Edition](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.15---oct-24--2024) and supports its protocols and drivers.
+
+## Release Highlights
+
+!!! important
+
+    This release of Percona Server for MongoDB includes a fix for a security vulnerability [CVE-2024-10921](https://nvd.nist.gov/vuln/detail/CVE-2024-10921). This vulnerability allowed an authorized user to trigger server crashes or receive the contents of the buffer over-reads of the server memory by sending specially crafted requests that constructed malfolmed BSON in MongoDB. The issue is fixed upstream in versions 6.0.18 and 7.0.15. The fix is also included in Percona Server for MongoDB  6.0.18-16 and 7.0.15-9.
```
Comment (radoslawszulgo) [inline docs/release_notes/7.0.15-9.md]:
```suggestion
    This release of Percona Server for MongoDB includes a fix for a security vulnerability [CVE-2024-10921](https://nvd.nist.gov/vuln/detail/CVE-2024-10921). This vulnerability allowed an authorized user to trigger server crashes or receive the contents of the buffer over-reads of the server memory by sending specially crafted requests that constructed malformed BSON in MongoDB. 
```

File: docs/release_notes/7.0.15-9.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Percona Server for MongoDB 7.0.15-9 includes the improvements and bug fixes of [MongoDB 7.0.15 Community Edition](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.15---oct-24--2024) and supports its protocols and drivers.
+
+## Release Highlights
+
+!!! important
+
+    This release of Percona Server for MongoDB includes a fix for a security vulnerability [CVE-2024-10921](https://nvd.nist.gov/vuln/detail/CVE-2024-10921). This vulnerability allowed an authorized user to trigger server crashes or receive the contents of the buffer over-reads of the server memory by sending specially crafted requests that constructed malfolmed BSON in MongoDB. The issue is fixed upstream in versions 6.0.18 and 7.0.15. The fix is also included in Percona Server for MongoDB  6.0.18-16 and 7.0.15-9.    
+
+    Users running any minor version of Percona Server for MongoDB 6.0.x before 6.0.18 and Percona Server for MongoDB 7.0.x before 7.0.15 should upgrade to Percona Server for MongoDB 6.0.18-16 and 7.0.15-9, respectively, as soon as possible.
```
Comment (radoslawszulgo) [inline docs/release_notes/7.0.15-9.md]:
```suggestion
    Users running any minor version of Percona Server for MongoDB 7.0.x before 7.0.15-9 should upgrade to this version as soon as possible. 
```

**Review by radoslawszulgo — CHANGES_REQUESTED** [review]
Please also add a reference to the "upgrade to minor version" page that mention adjustments in configuration file (fork: true)

File: docs/release_notes/7.0.15-9.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* [SERVER-95279](https://jira.mongodb.org/browse/SERVER-95279) - Use a new C++ type for BSON field names to ensure validity
+
+Find the full list of changes in the [MongoDB 7.0.15 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.15---oct-24--2024).
+
+## Changelog
+
+### Improvements
+
+* [PSMDB-1441](https://perconadev.atlassian.net/browse/PSMDB-1441) - Fixed the issue with master encryption keys getting lost when the number of created secrets exceeds the threshold by preventing a new secret creation and alerting users about it.
+* [PSMDB-1539](https://perconadev.atlassian.net/browse/PSMDB-1539) - Removed the fork:true and pidfile management to simplify migration
```
Comment (igorsol) [inline docs/release_notes/7.0.15-9.md]: This fix was reverted by https://github.com/percona/percona-server-mongodb/pull/1346

**Review by igorsol — CHANGES_REQUESTED** [review]

File: variables.yml [hunk 4]
```diff
@@ -2,10 +2,11 @@
 # See also mkdocs.yml plugins.with-pdf.cover_subtitle and output_path
 
 
-release: '7.0.14-8'
+release: '7.0.15-9'
 version: '7.0'
-mongosh: '2.2.10'
+mongosh: '2.3.2024-09-23'
```
Comment (sandraromanchenko) [inline variables.yml]:
```suggestion
mongosh: '2.3.2'
```

**Review by sandraromanchenko — APPROVED** [review]

File: docs/release_notes/7.0.15-9.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+Percona Server for MongoDB 7.0.15-9 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB Community Edition.
+
+Percona Server for MongoDB 7.0.15-9 includes the improvements and bug fixes of [MongoDB 7.0.15 Community Edition](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.15---oct-24--2024) and supports its protocols and drivers.
+
+## Release Highlights
+
+!!! important
+
+    This release of Percona Server for MongoDB includes a fix for a security vulnerability [CVE-2024-10921](https://nvd.nist.gov/vuln/detail/CVE-2024-10921). This vulnerability allowed an authorized user to trigger server crashes or receive the contents of the buffer over-reads of the server memory by sending specially crafted requests that constructed malformed BSON in MongoDB. he issue is fixed and included in Percona Server for MongoDB 7.0.15-9.
```
Comment (sandraromanchenko) [inline docs/release_notes/7.0.15-9.md]:
```suggestion
    This release of Percona Server for MongoDB includes a fix for a security vulnerability [CVE-2024-10921](https://nvd.nist.gov/vuln/detail/CVE-2024-10921). This vulnerability allowed an authorized user to trigger server crashes or receive the contents of the buffer over-reads of the server memory by sending specially crafted requests that constructed malformed BSON in MongoDB. The issue is fixed and included in Percona Server for MongoDB 7.0.15-9.    
```

**Review by sandraromanchenko — APPROVED** [review]

**Review by radoslawszulgo — APPROVED** [review]

**Review by igorsol — APPROVED** [review]

**Pull request merged**